cdap-ui/app/cdap/components/CaskWizards/LicenseStep/index.js

```diff
@@ -30,9 +31,11 @@ export default class LicenseStep extends Component {
   }
   componentWillMount() {
     let { entityName, entityVersion, licenseFileName } = this.props;
+    const marketHost = MarketStore.getState().selectedMarketHost;
     let params = {
       entityName,
       entityVersion,
+      marketHost,
```

Comment:
This might look a little unconventional, we are passing the path as main url and the host as query parameter. We can define the market host to be a url param, so that when we make a call to the Nodejs it already has the full url to talk to.

For instance in api/market.js we can define,
```
{
...
getSampleData: apiCreateorAbsPath(
  dataSrc,
  'GET',
  'REQUEST',
  ':host/packages/:entityName/:entityVersion/:filename',
   requestOptions
),
...
```

This still makes the call to `getSampleData` the same, but the host is actually added and we send a valid market url.

Reply:
Switched to use :marketHost